exporters/metric/dogstatsd/aggregation/distribution/distribution.go

```diff
@@ -0,0 +1,194 @@
+package distribution
+
+// lifted from import "go.opentelemetry.io/otel/sdk/metric/aggregator/array"
+
+import (
+	"context"
+	"math"
+	"sort"
+	"sync"
+	"unsafe"
+
+	"go.opentelemetry.io/otel/api/core"
+	"go.opentelemetry.io/otel/api/metric"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregator"
+)
+
+type (
+	Aggregator struct {
+		// ckptSum needs to be aligned for 64-bit atomic operations.
+		ckptSum    core.Number
+		lock       sync.Mutex
+		current    points
+		checkpoint points
+	}
+
+	points []core.Number
+)
+
+var _ export.Aggregator = &Aggregator{}
+var _ aggregator.MinMaxSumCount = &Aggregator{}
+var _ aggregator.Distribution = &Aggregator{}
+var _ aggregator.Points = &Aggregator{}
```

Comment:
I assume we can't use the OTel ddsketch aggregator b/c it doesn't implement `aggregator.Points`. Is this accurate?

Reply:
Aye. The whole point of a sketch is to avoid stashing an array of points.  If Datadog supported a sketch serialization wire format we could in theory write that to the Datadog API instead.